src/components/dialogs/create-contingency-list-dialog.js

```diff
@@ -54,6 +66,14 @@ export const CreateContingencyListDialog = ({ open, onClose }) => {
     const classes = useStyles();
     const intl = useIntl();
     const timer = React.useRef();
```

Comment:
Why a timer here ? to check if name exist ? why not using the already coded custom hook useNameField which do this ? code duplication

Reply:
Maybe in another PR?